SofaKernel/modules/SofaBaseMechanics/BarycentricMapping.h

```diff
@@ -252,13 +252,15 @@ class TopologyBarycentricMapper : public BarycentricMapper<In,Out>
     virtual void resize( core::State<Out>* toModel ) = 0;
 
 protected:
-    TopologyBarycentricMapper(core::topology::BaseMeshTopology* fromTopology, topology::PointSetTopologyContainer* toTopology = NULL)
-        : fromTopology(fromTopology), toTopology(toTopology)
+	TopologyBarycentricMapper(core::topology::BaseMeshTopology* fromTopology, topology::PointSetTopologyContainer* toTopology = NULL, bool useRestPosition = false)
+		: fromTopology(fromTopology), toTopology(toTopology), useRestPosition(useRestPosition)
```

Comment:
The change in this file are independent to CM topo and non breaking. So maybe merged through cherry picking in a PR ?